src/libmongoc/src/mongoc/mongoc-openssl-private.h

```diff
@@ -21,8 +21,6 @@
 
 #include <mongoc/mongoc-stream-tls-openssl-private.h>
 
-#include <mongoc/mongoc-ssl.h>
-
```

Comment:
Is this removal necessary/intentional given the now-guarded include in test-mongoc-topology-scanner.c?

Reply:
Accidentaly